layouts/partials/implementations.html

```diff
@@ -44,12 +43,24 @@ <h5 class="content-center">Desktop App</h5>
           <div class="app-ss content-center">
             <img src="/images/ipfs-companion-ss.png" />
           </div>
-          <h5 class="content-center">Browser Companion</h5>
-          <p align="left">Upgrade your browser with IPFS super powers. This add-on enables everyone to access IPFS resources the way they
-            were meant to.</p>
+          <h5 class="content-center">Browser Extension</h5>
+          <p align="left">
+            Upgrade your browser with IPFS superpowers. The IPFS Companion Web Extension connects your local IPFS daemon with your browser, enabling ipfs:// scheme urls and more!
+          </p>
         </a>
       </div>
     </div>
 
+    <div class="grid-flex-container" style="padding-top: 30px;">
+      <div class="grid-flex-cell grid-flex-cell-1of2 content-center">
+        <a href="https://cluster.ipfs.io/" target="_blank">
+          <div class="content-center">
+            <img src="/images/ipfs-cluster.png" />
+          </div>
+          <h5 class="content-center">IPFS Cluster</h5>
+          <p align="left">Ideal for large datasets or servers, IPFS Cluster automatically allocates, replicates, and tracks your data as pinsets across multiple IPFS nodes. It's both a standalone application and a CLI client.</p>
```

Comment:
`large-scale` --> `large` (same meaning, but more concise)
`multiple servers` --> `servers` (to @olizilla's point, Cluster is useful even when running one server with one IPFS node.
Added `automatically` to emphasize the automagic nature of what Cluster does. I'm not sure this is really needed; feel free to remove.

Reply:
....and now I'm seeing that this needs another edit. `large datasets or servers` should be switched to `servers or large datasets` so the reader doesn't see `large datasets or large servers` as the meaning. One moment, hot commit coming in...